mod/events.php

```diff
@@ -96,7 +93,7 @@ function events_post(App $a) {
 	$action = ($event_id == '') ? 'new' : "event/" . $event_id;
 	$onerror_url = App::get_baseurl() . "/events/" . $action . "?summary=$summary&description=$desc&location=$location&start=$start_text&finish=$finish_text&adjust=$adjust&nofinish=$nofinish";
 
-	if (strcmp($finish,$start) < 0 && !$nofinish) {
+	if (strcmp($finish, $start) < 0 && !$nofinish) {
 		notice( t('Event can not end before it has started.') . EOL);
 		if (intval($_REQUEST['preview'])) {
 			echo( t('Event can not end before it has started.'));
```

Comment:
Standards: Please remove parentheses from language constructs. (also works for `include`/`require`, `return`, etc...)